tests/queries/0_stateless/01455_leaf_max_rows_bytes_to_read.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+. "$CURDIR"/../shell_config.sh
+
+$CLICKHOUSE_CLIENT --query="DROP TABLE IF EXISTS test_local;"
+$CLICKHOUSE_CLIENT --query="DROP TABLE IF EXISTS test_distributed;"
+
+$CLICKHOUSE_CLIENT --query="CREATE TABLE test_local (date Date, value UInt32) ENGINE = MergeTree(date, date, 8192);"
+$CLICKHOUSE_CLIENT --query="CREATE TABLE test_distributed AS test_local ENGINE = Distributed(test_shard_localhost, currentDatabase(), test_local, rand());"
+$CLICKHOUSE_CLIENT --query="INSERT INTO test_local SELECT '2000-08-01', number as value from numbers(50000);"
+
+# Must throw an exception
+EXCEPTION_TEXT="Limit for rows (controlled by 'max_rows_to_read_leaf' setting) exceeded"
```

Comment:
Let's add a test for `max_rows_to_read`/`max_bytes_to_read` (since this PR touches it too)

Reply:
Replaced the .sh test with .sql one so only exception codes are checked now.